src/py/mains/ovlp_stats.py

```diff
@@ -44,14 +42,13 @@ def filter_stats(readlines, min_len):
             if idt < 90:
                 continue
 
-            if not contained:
-                if l[-1] in ("contains", "overlap"):
-                    ave_idt += idt * overlap_len
-                    all_over_len += overlap_len
-                if q_s == 0:
-                    overlap_data["5p"] += 1
-                elif q_e == q_l:
-                    overlap_data["3p"] += 1
+            if l[-1] in ("contains", "overlap"):
+                ave_idt += idt * overlap_len
+                all_over_len += overlap_len
+            if q_s == 0:
+                overlap_data["5p"] += 1
+            if q_e == q_l:
```

Comment:
Oh, this is the real change, `if` instead of `elif`. I see now. I wasn't sure what these counts meant.